sources/mysql/infoschema.go

```diff
@@ -219,20 +219,39 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 		} else {
 			colAutoGen = ddl.AutoGenCol{}
 		}
+		defaultVal := schema.DefaultValue{
+			IsPresent: colDefault.Valid,
+			Value:     "",
+		}
+		if colDefault.Valid {
+			defaultVal.Value = sanitizeDefaultValue(colDefault.String)
+		}
 		c := schema.Column{
-			Id:      colId,
-			Name:    colName,
-			Type:    toType(dataType, columnType, charMaxLen, numericPrecision, numericScale),
-			NotNull: common.ToNotNull(conv, isNullable),
-			Ignored: ignored,
-			AutoGen: colAutoGen,
+			Id:           colId,
+			Name:         colName,
+			Type:         toType(dataType, columnType, charMaxLen, numericPrecision, numericScale),
+			NotNull:      common.ToNotNull(conv, isNullable),
+			Ignored:      ignored,
+			AutoGen:      colAutoGen,
+			DefaultValue: defaultVal,
 		}
 		colDefs[colId] = c
 		colIds = append(colIds, colId)
 	}
 	return colDefs, colIds, nil
 }
 
+// SanitizeDefaultValue removes extra characters added to Default Values.
+// Example:
+// Default Value: "concat('John', 'swamp', 'span')"
+// Default Value with extra characters: "concat(_utf8mb4\\'John\\',_utf8mb4\\'swamp\\',_utf8mb4\\'span\\')"
+func sanitizeDefaultValue(str string) string {
+	str = strings.ReplaceAll(str, "\\", "")          // Default Value after removing all backslashes: "concat(_utf8mb4'John',_utf8mb4'swamp',_utf8mb4'span')"
+	after := strings.Replace(str, "_utf8mb4", "", 1) // Default Value after removing first "_utf8mb4": "concat('John',_utf8mb4'swamp',_utf8mb4'span')"
```

Comment:
Just club this into one, because what it the first value doesn't have _utf8mb4?
Also please rename "str" to defaultValueString or something more descriptive

Reply:
Ideally this could be any collation like _utf8mb3 or _ucs2_croatian_ci or anything else for that matter.
It might be ok if we craft a regex to  `_<word>'<literal>'` pattern, but we need to be super careful that we are not having un-intended side-effects.

In the absence of a good source Dialect to Spanner-Dialect translation, it might be best to let the user edit this IMHO.